docs/how-to-guides/new-user-guides/authentication-permissions-and-global-configuration/pod-security-standards.md

```diff
@@ -2,21 +2,74 @@
 title: Pod Security Standards
 ---
 
+[Pod Security Standards](https://kubernetes.io/docs/concepts/security/pod-security-standards/) define
+security restrictions for a broad set of workloads. 
+They replace [Pod Security Policies](https://kubernetes.io/docs/concepts/security/pod-security-policy/),
+which are now deprecated.
```

Comment:
```suggestion
[Pod Security Standards](https://kubernetes.io/docs/concepts/security/pod-security-standards/) define security restrictions for a broad set of workloads.  They replace [Pod Security Policies](https://kubernetes.io/docs/concepts/security/pod-security-policy/), which are now deprecated.
```

Please update other instances where there are line breaks in the middle of sentences. Unfortunately, the line breaks get rendered to the published page.

Reply:
Ah, ok.  Sorry about that.  I usually default to keeping lines short.